1228-public_page/anonymous_user_views_landing_page_for_registration.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Anonymous User views Landing Page for Registration
+
+    @881
+    Scenario Outline: Anonymous User views Landing Page for Registration
+        Given the Anonymous User has opened NVA
+        And they see the "<Page>"
+        When they click to view a Registration
+        Then the Landing Page for Registration is opened
+        And they see page fields for
+            | Title                           |
+            | Abstract                        |
+            | NPI                             |
+            | Keywords                        |
+            | Publication date                |
+            | Primary language                |
+            | Projects                        |
+            | Publication type                |
+            | Publication subtype             |
+            | Fields corresponding to subtype |
+            | Contributors                    |
+            | Files                           |
+            | DOI link                        |
+            | License                         |
+        Examples:
+            | Page        |
+            | Start Page  |
+            | Public Page |
```

Comment:
Trenger vi egentlig `<Page>`? skjønner ikke helt hva det er...
```suggestion
    Scenario: Anonymous User views Landing Page for Registration
        Given the Anonymous User has opened NVA
        When they navigate to Landing Page for a Registration
        Then they see page fields for
            | Title                           |
            | Abstract                        |
            | NPI                             |
            | Keywords                        |
            | Publication date                |
            | Primary language                |
            | Projects                        |
            | Publication type                |
            | Publication subtype             |
            | Fields corresponding to subtype |
            | Contributors                    |
            | Files                           |
            | DOI link                        |
            | License                         |
```

Reply:
hehe, godt poeng! Støtter fjerning. Husk å omdøpe til `Scenario` og ikke `Scenario Outline` også da

Reply:
Er det publication type eller registration type?

Reply:
reg 👍

Reply:
eller bruken er sikkert 50/50 i dag, men jeg mener registration

Reply:
kan vel egentlig bare fjerne type også, for vi viser jo bare subtype